### PR TITLE
Removed 'View a list of all the added/changed records' for my annotat…

### PR DESCRIPTION
**Pull request by alexhuang091**
…ion alert

### CODE REVIEW

File: grails-app/views/email/biocache.gsp [hunk 1]
```diff
@@ -7,8 +7,8 @@
     <p>
       <g:message code="${message}" default="${message}" args="${[totalRecords]}"/>
     </p>
-    <p><a href="${moreInfo}"><g:message code="biocache.view.list" /></a></p>
-
+    <g:set var="listcode" value="${(g.message(code:"query.myannotations.title", default:"My Annotations") == query.name) ? "biocache.view.myannotation.list" : "biocache.view.list"}"/>
+    <p><a href="${moreInfo}"><g:message code="${listcode}"/></a></p>
```
[the remark this answers is not in the document]
Comment (alexhuang091) [reply grails-app/views/email/biocache.gsp:11]: Hi Rita, no change needed for `moreInfo`, only the link text 

截屏2021-04-06 下午3 28 48

File: grails-app/views/email/biocache.gsp [hunk 2]
```diff
@@ -7,8 +7,8 @@
     <p>
       <g:message code="${message}" default="${message}" args="${[totalRecords]}"/>
     </p>
-    <p><a href="${moreInfo}"><g:message code="biocache.view.list" /></a></p>
-
+    <g:set var="listcode" value="${(g.message(code:"query.myannotations.title", default:"My Annotations") == query.name) ? "biocache.view.myannotation.list" : "biocache.view.list"}"/>
```
Comment (Rita-C) [inline grails-app/views/email/biocache.gsp:10]: hmm, feel  `listcode` should be decided in `.groovy`, this gsp is the view, does not need to know business logic. Example at `QueryController`

Comment (alexhuang091) [reply grails-app/views/email/biocache.gsp:10]: ok, let me have a look

File: grails-app/views/email/biocache.gsp (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rita-C) [reply grails-app/views/email/biocache.gsp:11]: "View records you have annotated " on https://auth-test.ala.org.au/userdetails/myprofile/ is the link I was referring to (different URL compares to your screenshot).

Does it go to the same page?

[the remark this answers is not in the document]
Comment (alexhuang091) [reply grails-app/views/email/biocache.gsp:11]: yes, they look different but go to the same page.

I'll make them exactlly same.

**Review by Rita-C — APPROVED** [review]

**Review by Rita-C — APPROVED** [review]
Looks great, thanks.

**Pull request merged**